lib/src/group_changes.dart

```diff
@@ -32,12 +32,36 @@ class MinMax {
 /// sets as keys in a later map, so we use the keys, and can recover the sets.
 Map<String, List<String>> configSets;
 
+class Result {
+  Result(Map<String, dynamic> data)
+      : previous = data['previous_result'],
+        current = data['result'],
+        expected = data['expected'],
+        // Different methods of accessing BigQuery give different types here.
+        matches = data['matches'] || data['matches'] == 'true',
+        key = [data['previous_result'], data['result'], data['expected']]
+            .join(' ');
```

Comment:
Would string interpolation be cleaner?

Reply:
I want to emphasise that it is just a list, concatenated with a separator, not a formatted, humanly readable toString.

Reply:
Actually, it is not used at all except in hashCode and operator==.  
Removing it.